themes/default/content/docs/pulumi-cloud/oidc/client/kubernetes-gke.md

```diff
@@ -0,0 +1,133 @@
+---
+title_tag: Configure OpenID Connect for Google Kubernetes Engine | OIDC
+meta_desc: This page describes how to configure Pulumi to accept Google Kubernetes Engine OIDC tokens
+title: Google Kubernetes Engine
+h1: Configuring OpenID Connect for Google Kubernetes Engine
+meta_image: /images/docs/meta-images/docs-meta.png
+menu:
+    pulumicloud:
+        parent: openid-connect-client
+        weight: 1
+---
+
+This document outlines the steps required to configure Pulumi to accept Google Kubernetes Engine id_tokens to be exchanged by Organization access tokens
+
+## Prerequisites
+
+* You must be an admin of your Pulumi organization.
+
+{{< notes type="warning" >}}
+Please note that this guide provides step-by-step instructions based on the official provider documentation which is subject to change. For the most current and precise information, always refer to the [official Bound Service Account Tokens documentation](https://github.com/kubernetes/enhancements/tree/master/keps/sig-auth/1205-bound-service-account-tokens).
```

Comment:
Consider using this link to Kubernetes user documentation, rather than pointing to the underlying design doc.
https://kubernetes.io/docs/concepts/storage/projected-volumes/#serviceaccounttoken